### ISSUE TITLE
Colour contrast for govt agencies is insufficient

### ISSUE THREAD

**Issue by timClicks** [ccnz/Ushahidi_Web#97]
"Hey Tim, the colour for Govt agencies on eq.org.nz map doesnt have good contrast -- reckon this could be fixed?"

See http://twitter.com/#!/_Nathan_W_/status/42306747238580225

**Comment by Br3nda** [comment]
They are now PINK
